client/package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Avocano app",
   "license": "MIT",
   "author": "avocano-shell",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
Ideally, this version should match `/version.txt`. There may be a way to have release-please update both, but please don't introduce another version if possible. We already have the version of this repo, and the version of terraform-dynamic-python-webapp

Reply:
@grayside I hear you know how to ask release-please nicely how to do this (or know where to find out how)

Reply:
I have not done this, but what we can do is use [Manifest Releaser](https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md) so the release-please PR will release the overall repository as well as targeted directories within the repo.

I will experiment with this tomorrow. The approach will result in writing the avocano version to the package.json as part of the release-please PR.

Reply:
PR in #168